acis_thermal_check/main.py

```diff
@@ -715,6 +716,7 @@ def make_validation_plots(self, tlm, model_spec, outdir, run_start):
                         ax.axhline(self.plan_limit_lo, linestyle='--', color='y')
                         ymin = min(self.yellow_lo-1, ymin)
                 ax.set_ylim(ymin, ymax)
+            ax.set_xlim(xmin, xmax)
```

Comment:
@jzuhone - for my benefit in future Py3 migration, do you understand this change in behavior?  Presumably when you did this with Ska3 / MPL 2+, the X axis limits were different?